runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/ExecutionStateTracker.java

```diff
@@ -321,6 +321,11 @@ public long getNextLullReportMs() {
     return nextLullReportMs;
   }
 
+  /** Return the time of the next bundle lull report. */
```

Comment:
This sounds like an absolute time, while this is the duration after the bundle start that the log should occur.  Maybe better name would be nextBundleLullDurationReportMs for method and variable?

this is just exposed for testing as well so you could remove the public and have it be package private if that works. or mark with the @VisibleForTesting annotaiton.

Reply:
Done.